### PR TITLE
Cloudinary-SDK_CU-mhdnmj

### PR DESCRIPTION
**Pull request by a-wilkes**

### CODE REVIEW

**Comment by a-wilkes** [comment]
Task linked: [CU-mhdnmj Cloudinary SDK](https://app.clickup.com/t/mhdnmj)

**Comment by a-wilkes** [comment]
I tried to do the plugins as well, but I couldn't get the component that Cloudinary supposedly imports working. I found it referred to in the documentation as `<advanced-image>`, `<cl-img>`, `<cld-img>`, and `<cl-image>`, but all of them returning an error when I tried to use them (though it does seem to work in their [example project](https://codesandbox.io/s/angular-sdk-sample-95juh?file=/src/app/app.component.html), so maybe it's an issue with imports?).

File: src/app/shared/models/classes/product.class.ts [hunk 1]
```diff
@@ -9,6 +9,7 @@ export abstract class Product {
     collection: CollectionType | null = null;
     description: string | null = null;
     slug: string | null = null;
+    images: { main: string | null, thumb: string | null } = { main: null, thumb: null };
```
Comment (lpboucher) [inline src/app/shared/models/classes/product.class.ts]: let's add a custom type for product image

File: src/app/modules/products/components/product-card/product-card.component.html [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <div class="product-card">
     <div class="product-card-image-container">
-        <img class="product-card-image" src="https://images.unsplash.com/photo-1475855581690-80accde3ae2b?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=750&q=80" alt="">
+        <img class="product-card-image" src="{{imageUrl}}" alt="">
```
Comment (lpboucher) [inline src/app/modules/products/components/product-card/product-card.component.html]: prefer to use [] bindings -> [src]="imageUrl"

**Review by lpboucher — APPROVED** [review]

File: src/app/shared/models/classes/product.class.ts (hunk 1 above)
Comment (lpboucher) [reply src/app/shared/models/classes/product.class.ts]: done

File: src/app/modules/products/components/product-card/product-card.component.html (hunk 2 above)
Comment (lpboucher) [reply src/app/modules/products/components/product-card/product-card.component.html]: done

**Pull request merged**